pallets/score-staking/src/lib.rs

```diff
@@ -491,14 +444,31 @@ pub mod pallet {
 						return Err(Error::<T>::TokenStakingAmountAlreadyUpdated.into());
 					}
 					let last_round = Round::<T>::get();
-					s.last_round_reward += amount;
```

Comment:
```
let last_round = Round::<T>::get();
```

The round_interval is configurable, what if `Round::<T>::get()` is bigger than `round_index`? Shall we always use `round_index` here to assign `s.last_token_distributed_round`?